pipeline/scripts/inference_pipeline.jl

```diff
@@ -1,38 +1,46 @@
 using DrWatson
 @quickactivate "Analysis pipeline"
 
+# Include the AnalysisPipeline module
 include(srcdir("AnalysisPipeline.jl"))
+# Include the common parameter values
 include(scriptsdir("common_param_values.jl"))
+# Include the common inference scenarios
+include(scriptsdir("common_scenarios.jl"))
 
 @info("""
       Running inference on truth data.
-
       ---------------------------------------------
       Currently active project is: $(projectname())
       Path of active project: $(projectdir())
-      """)
-
-## Set up the truth Rt and save a plot of it
-# Set up the EpiAware models to use for inference.
-using .AnalysisPipeline, Plots, JLD2, EpiAware, Distributions
-
-#Common priors for initial process and std priors
-transformed_process_init_prior = Normal(0.0, 0.25)
-underlying_std_prior = HalfNormal(1.0)
 
-ar = AR(damp_priors = [Beta(0.5, 0.5)], std_prior = underlying_std_prior,
-    init_priors = [transformed_process_init_prior])
-
-rw = RandomWalk(
-    std_prior = underlying_std_prior, init_prior = transformed_process_init_prior)
-
-diff_ar = DiffLatentModel(; model = ar, init_priors = [transformed_process_init_prior])
-
-wkly_ar, wkly_rw, wkly_diff_ar = [ar, rw, diff_ar] .|>
-                                 model -> BroadcastLatentModel(model, 7, RepeatBlock())
+      """)
 
-## Parameter settings
-# Rolled out to a vector of inference configurations using `dict_list`.
-sim_configs = Dict(:igp => [DirectInfections, ExpGrowthRate, Renewal],
-    :latent_model => [wkly_ar, wkly_rw, wkly_diff_ar],
-    :gi_mean => gi_means, :gi_std => gi_stds) |> dict_list
+## Inference methods
+using Distributions, .AnalysisPipeline, EpiAware, JLD2, ADTypes, AbstractMCMC
+
+for filename in readdir(datadir("truth_data"))
```

Comment:
This is where dagger comes in? Or we would do a dagger based overhaul of the whole pipeline?